source/maintenance-guide/maintenance-guide.rst

```diff
@@ -178,6 +178,33 @@ More information can be found at https://www.chirpstack.io/application-server/in
 
 For installation configuration of Chirpstack see: https://www.chirpstack.io/application-server/install/config/
 
+Migrations
+-----------
+
+This project is using TypeORM migrations when changes are applied to the database. This is recommended by TypeORM when the project is used for production.
+
+Generate migrations
+~~~~~~~~~~~~~~~~~~~
+
+If you make some changes in the database you need to generate the migration. In the console you write: "npm run generate-migration <your name of the migration>". Then a migration file 
+will be created in a Migration folder. A timestamp will be added to the name to indicate when the migration has been generated. If no changes are made to the database, nothing will happen.
+
+Run migrations
+~~~~~~~~~~~~~~~~
+When the project is starting, a new command will be called automatically. This happens every time you run the program because of a prestart script.
+The command is run-migrations which runs all the pending generated migrations in the Migrations folder, starting from the oldest migration. If there is no pending migrations then nothing will happen.
+
+It will happen in both debug and prod mode.
+
+Revert migration
+~~~~~~~~~~~~~~~~~~
+If you later on wish to revert a migration you can write npm run migration:revert. Then the latest runned migration will be reverted. You can continue to do this until you reach the desired migration.
```

Comment:
Commands should be formatted. Also, do you mean `npm run typeorm migration:revert`?